clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -143,15 +143,29 @@ public FetchResponse(Struct struct) {
     public Send toSend(String dest, RequestHeader requestHeader) {
         ResponseHeader responseHeader = new ResponseHeader(requestHeader.correlationId());
 
+        boolean hasThrottleTime = struct.hasField(THROTTLE_TIME_KEY_NAME);
```

Comment:
Oh, I see what you mean. I removed this commit from this PR since it's a bit controversial and it's part of a wider conversation about maintainability and avoiding unnecessary allocation. I will probably include it in the next PR along with other items that need discussion.